src/pages/index.js

```diff
@@ -24,10 +46,123 @@ const profileEditButton = document.querySelector("#profile-edit-button");
 const profileCardModal = document.querySelector("#profile-card-modal");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const profileEditForm = profileEditModal.querySelector("#editProfileForm");
+const saveButton = document.querySelector(".modal__save-button");
```

Comment:
There are 3 modals in the project with buttons and 2 forms. You need to **name** variables and functions the way all could understand what they are. 


`profileSaveButton`


**Please, correct it everywhere**